include/arch/arm/cortex_m/scripts/linker.ld

```diff
@@ -515,6 +515,10 @@ SECTIONS
 #include <custom-sections.ld>
 #endif
 
+/* Located in generated directory. This file is populated by the
+ * add_named_section() Cmake function. */
+#include <generated-sections.ld>
+
```

Comment:
We need to add this to all linker.ld unfortunately.

Reply:
Will do